README.md

```diff
@@ -280,7 +292,469 @@ Here are a few examples on how to use the SDK. If you have any doubts, use the b
 `help()` function to get more info on the desired functionality
 (for example: `help(starkinfra.boleto.create)`)
 
-## Create pix requests
+## Issuing
+
+### Process Authorizations
+
+It's easy to process Authorizations delivered to your Webhook endpoint. 
+
+Your endpoint must respond the request up to 2 seconds. Otherwise, the authorization will be denied.
```

Comment:
If you do not approve or decline the authorization within 2 seconds, the authorization will be denied.